src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -117,6 +110,13 @@ public static void LargestWindowWidth_UnixGet_ReturnsExpected()
         Helpers.RunInRedirectedOutput((data) => Assert.Equal(Console.WindowWidth, Console.LargestWindowWidth));
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS)]  // Expected behavior specific to Unix
```

Comment:
I can't see `OSX` and `FreeBSD` here, does it mean that the method is now supported on OSX and BSD? If so, you should add more `SupportedOSPlatform` attributes to the ref assembly.

Reply:
Thank you. `FreeBSD` and `OSX` are added to `SupportedOSPlatform`.